exercises/hangman/HangmanTest.cs

```diff
@@ -1,135 +1,342 @@
-﻿using Xunit;
-
-public class HangmanTest
+﻿using System;
+using Xunit;
+using Microsoft.Reactive.Testing;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Reactive.Concurrency;
+
+public class HangmanTests: ReactiveTest
 {
     [Fact]
-    public void Initially_9_failures_are_allowed()
+    public void Initial_state_masks_the_word()
     {
-        var game = new HangmanGame("foo");
+        // Arrange
+        Hangman hangman = new Hangman("foo");
+        string actual = "";
+
+        // Act
+        // to learn more about observers check out http://reactivex.io/documentation/observable.html
```

Comment:
I would suggest to move the general information on Rx to the README. To do that, you'll have to update the [HINTS.md] file(https://github.com/exercism/csharp/blob/master/exercises/hangman/.meta/hints.md). Then you have to update the README.